pkgs/tools/graphics/lsix/default.nix

```
@@ -1,17 +1,17 @@
-{ lib, stdenvNoCC, fetchFromGitHub, makeWrapper, imagemagick }:
+{ lib, stdenvNoCC, fetchFromGitHub, makeBinaryWrapper, imagemagick }:
 
-stdenvNoCC.mkDerivation rec {
+stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "lsix";
   version = "1.8.2";
 
   src = fetchFromGitHub {
     owner = "hackerb9";
-    repo = pname;
-    rev = version;
+    repo = finalAttrs.pname;
```

This pattern should never be used, `pname` and `repo` are independent.

```suggestion
    repo = "lsix";
```

---

I believe i've fixed the changes you requested